README.md

```diff
@@ -12,21 +13,24 @@ See full conversion procedure in sections below.
 This Lambda currently supports the following conversions:
 
 | Source (DynamoDB)   | Destination (Aurora) |
-|---------------------|----------------------|
+| ------------------- | -------------------- |
```

Comment:
If we are going to say which DynamoDB tables this processes - it needs to include Activities!